src/views/Scan/AdviceScreen.vue

```diff
@@ -0,0 +1,109 @@
+// This program has been developed by students from the bachelor Computer Science at 
+// Utrecht University within the Software Project course.
+
+// © Copyright Utrecht University (Department of Information and Computing Sciences)
+
+<template>
+  <ion-page>
+    <ion-header>
+      <ion-toolbar>
+        <ion-button slot="start" router-direction="root" router-link="/home" fill="clear">
+          <ion-icon :icon="arrowBackOutline"></ion-icon>
+        </ion-button>
+        <ion-title>
+          Stappen plan scannen
+        </ion-title>
```

Comment:
Stappenplan is één woord
```suggestion
        <ion-title>
          Stappenplan scannen
        </ion-title>
```